web-dev-cody-hackathon/100editors
# add dynamic routing and move DocumentWrapper to room

# Description
Add dynamic routes so users can make changes in a particular room.

For example: `{url}/room1` vs `{url}/room2`

The above `room1` and `room2` are different rooms with data being shared between the users in the room, but different across different rooms

## Testing

- Go to `/one` and `/two`  (or any other dynamic route)
- type something in the editor in room `one`, the data should be shared for everyone in that room
- type something else in room `two`, the data should be shared for everyone in room `two`

## Type of change

- New feature (non-breaking change which adds functionality)


## Screenshots

<img width="974" alt="image" src="https://github.com/web-dev-cody-hackathon/100editors/assets/45009203/2accc3ee-d66d-47c3-a32a-81be183820ca">

<img width="1001" alt="image" src="https://github.com/web-dev-cody-hackathon/100editors/assets/45009203/d1085d97-bb16-4bf3-9aca-cef12fbeefb3">


# Checklist:

- [ ] Changes have new/updated automated tests, if applicable
- [ ] Changes have new/updated docs, if applicable
- [ ] I have added comments on any new, hard-to-understand code
- [ ] Changes generate no new errors or warnings
